test/docfx.Test/DocfxTest.cs

```diff
@@ -206,30 +235,67 @@ private static async Task RunLanguageServer(string docsetPath, DocfxTestSpec spe
 
         private static void RunBuild(string docsetPath, string outputPath, bool dryRun, bool singleFile, DocfxTestSpec spec, Package package)
         {
-            var randomOutputPath = Path.ChangeExtension(outputPath, $".{Guid.NewGuid()}");
+            var randomNormalOutputPath = Path.ChangeExtension(outputPath, $".normal.{Guid.NewGuid()}");
+            var randomFirstStageOutputPath = Path.ChangeExtension(outputPath, $".firststage.{Guid.NewGuid()}");
+            var randomContinueOutputPath = Path.ChangeExtension(outputPath, $".continue.{Guid.NewGuid()}");
 
             docsetPath = Path.Combine(docsetPath, spec.Cwd ?? "");
 
+            var isOutputTypePageJson = IsOutputTypePageJson(spec);
+            var isTwoStagesBuild = isOutputTypePageJson && !dryRun && !spec.NoContinue;
```

Comment:
Update `ExpandTest` to expand `ContinueBuild` into a new test case. Now 1 stage build of `outputType: pagejson` tests are not running.